### PR TITLE
created separate comment sections for each blog post

### PR DESCRIPTION
**Pull request by christinachance**

### CODE REVIEW

File: portfolio/src/main/webapp/blog_script.js [hunk 1]
```diff
@@ -15,13 +15,19 @@
...
     fetch('/comments').then(response => response.json()).then((comments) =>{
-        const commentHistory = document.getElementById('oldComments');
-        comments.forEach((line) => {
-            commentHistory.appendChild(createCommentElement(line));
-        })
+        const numCommentSections = document.getElementsByClassName('oldComments').length;
+        for(var postId=1; postId<=numCommentSections; postId++){
+            const commentHistory = document.getElementById(postId);
+            comments.forEach((comment) => {
+                if(postId== comment.postId){
```
Comment (wulian1223) [inline portfolio/src/main/webapp/blog_script.js:22]: missing space between `==`

File: portfolio/src/main/webapp/blog_script.js [hunk 2]
```diff
@@ -15,13 +15,19 @@
...
+                    commentHistory.appendChild(createCommentElement(comment));
+                }
+            })
+        }
     });
```
Comment (wulian1223) [inline portfolio/src/main/webapp/blog_script.js:27]: How about
```
comments.forEach((comment) => {
    const commentHistory = document.getElementById(comment.postId);    
    commentHistory.appendChild(createCommentElement(comment));
})
```

Comment (christinachance) [reply portfolio/src/main/webapp/blog_script.js:27]: That is so much more straightforward. Than you, done.

File: portfolio/src/main/webapp/blog_script.js (hunk 1 above)
Comment (christinachance) [reply portfolio/src/main/webapp/blog_script.js:22]: done.

File: portfolio/src/main/webapp/blog_script.js (hunk 2 above)
Comment (christinachance) [reply portfolio/src/main/webapp/blog_script.js:27]: That is so much more straightforward. Thank you and done.

Comment (christinachance) [reply portfolio/src/main/webapp/blog_script.js:22]: I tried this but there is an issue with how it ends up accessing the new posts. It doesn't retrieve the posts in the data store.

File: portfolio/src/main/webapp/blog_script.js [hunk 3]
```diff
@@ -50,3 +50,16 @@ function deleteComment(comment){
...
+var navbar = document.getElementById("nav");
+var sticky = navbar.offsetTop;
+
+function myFunction() {
+  if (window.pageYOffset >= sticky) {
+    navbar.classList.add("sticky");
+  } else {
+    navbar.classList.remove("sticky");
+  }
+}
```
Comment (serzhk) [inline portfolio/src/main/webapp/blog_script.js:65]: Add new line at the end

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -33,3 +33,16 @@ function showSlides(){
...
+var navbar = document.getElementById("nav");
+var sticky = navbar.offsetTop;
+
+function myFunction() {
+  if (window.pageYOffset >= sticky) {
+    navbar.classList.add("sticky");
+  } else {
+    navbar.classList.remove("sticky");
+  }
+}
```
Comment (serzhk) [inline portfolio/src/main/webapp/script.js:48]: Add empty line at the end

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -38,7 +38,7 @@ body{
     margin-right: 120px;
     width: 650px;
     text-align: justify;
-    word-wrap: normal;
+    /* word-wrap: normal; */
```
Comment (serzhk) [inline portfolio/src/main/webapp/style.css:41]: Remove commented line

File: portfolio/src/main/webapp/blog_script.js (hunk 3 above)
Comment (christinachance) [reply portfolio/src/main/webapp/blog_script.js]: done.

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (christinachance) [reply portfolio/src/main/webapp/script.js]: done.

File: portfolio/src/main/webapp/style.css (hunk 5 above)
Comment (christinachance) [reply portfolio/src/main/webapp/style.css]: done.

**Pull request merged**